src/Microsoft.VisualStudio.AppDesigner/ApplicationDesigner/ApplicationDesignerWindowPane.vb

```diff
@@ -71,6 +71,9 @@ Namespace Microsoft.VisualStudio.Editors.ApplicationDesigner
                 'Must marshal ourselves as IUnknown wrapper
                 VSErrorHandler.ThrowOnFailure(WindowFrame.SetProperty(__VSFPROPID.VSFPROPID_ViewHelper, New UnknownWrapper(_viewHelper)))
 
+                'Don't reopen the designer during solution restore
+                VSErrorHandler.ThrowOnFailure(WindowFrame.SetProperty(__VSFPROPID5.VSFPROPID_DontAutoOpen, True))
```

Comment:
This works, but one question about it, presumably for @davkean: I was told that we share designers and editor factories with legacy for the app designer, so I don't understand how this regression came about. There is no reference to this property, or its equivalent RDT_DontAutoOpen property, in any of the legacy code, or pkgdef files, in the entire vsproject folder in the VS repo.